src/sql/workbench/services/notebook/browser/models/modelInterfaces.ts

```diff
@@ -533,7 +533,7 @@ export interface ICellModel {
 	sendChangeToNotebook(change: NotebookChangeType): void;
 	cellSourceChanged: boolean;
 	readonly savedConnectionName: string | undefined;
-	readonly attachments: nb.ICellAttachments;
+	attachments: nb.ICellAttachments;
```

Comment:
This should be `nb.ICellAttachments | undefined` if we're following what the actual implementation is saying. (and then also please update the _attachments property in the cell class)